rolling/release-build.yaml

```
@@ -16,7 +16,7 @@ notifications:
   - ros2-buildfarm-rolling@googlegroups.com
   maintainers: true
 sync:
-  package_count: 262
+  packages: [desktop]
```

We should keep the package count as well as the minimum packages. 

```suggestion
  package_count: 500
  packages: [desktop]
```

Clearly the current limit was way too low. For other distros i've usually advocated at keeping it about at 90% of the number of successfully released packages. For rolling it could be lower, but I don't think that we should be really expecting major regressions of 20% and if we do we can lower the count to enable the sync. 

If this makes sense this should be propagated below to.